docs-developer/deploying.md

```diff
@@ -44,3 +44,9 @@ When you're ready with a fix landed on `main`, you can push a new version to the
 
 You can find the Mozilla contacts about our deployment in [this Mozilla-only
 document](https://docs.google.com/document/d/16YRafdIbk4aFgu4EZjMEjX4F6jIcUJQsazW9AORNvfY/edit).
+
+# Deploying on a local nginx instance
+
+To deploy on nginx (without support for direct upload from the Firefox UI), run `yarn build-prod`
```

Comment:
Is it worth mentioning the Firefox preference `devtools.performance.recording.ui-base-url` to point to another URL?

Reply:
I'm not sure if uploading from Firefox works on nginx (and without the other steps mentioned in https://github.com/firefox-devtools/profiler/issues/3764#issuecomment-997934001). I didn't test it, since I'm using perf profiles, not Firefox ones. I'll leave it for someone else to test and document.

Reply:
I'm deeply sorry that I missed your message earlier.

I believe there's a slight misunderstanding about how the profiler UI works.

As a first quick remark, please remember that the profiler UI is just static files.

When capturing a profile from Firefox, the URL pointed by `devtools.performance.recording.ui-base-url` is opened. The webpage (static) requests the profile using a "WebChannel" and Firefox answers with the data. Some more messages are exchanged to gather symbols too. But until now everything stays local, and there's no "upload" happening.

An upload would happen only if and when the user presses the "upload" button at the top right.

So I don't see an obvious reason why this wouldn't work.

But there may be some checks about URLs that I forgot, so I agree we should test it fully before documenting it.